operators/gpu-operator-certified/v23.9.2/metadata/annotations.yaml

```diff
@@ -15,3 +15,15 @@ annotations:
 
   # Annotations to specify OCP versions compatibility.
   com.redhat.openshift.versions: v4.9-v4.15
+
+  # Infrastructure feature annotations
+  features.operators.openshift.io/disconnected: "true"
+  features.operators.openshift.io/fips-compliant: "false"
+  features.operators.openshift.io/proxy-aware: "true"
+  features.operators.openshift.io/tls-profiles: "false"
+  features.operators.openshift.io/token-auth-aws: "false"
+  features.operators.openshift.io/token-auth-azure: "false"
+  features.operators.openshift.io/token-auth-gcp: "false"
+  features.operators.openshift.io/cnf: "false"
+  features.operators.openshift.io/cni: "false"
+  features.operators.openshift.io/cni: "false"
```

Comment:
Sorry for the overloaded term of annotations, but these go in the CSV's `annotations` section. See [this](https://github.com/opdev/simple-demo-operator/blob/main/bundle/manifests/simple-demo-operator.clusterserviceversion.yaml#L20-L28) example for further clarity.

Reply:
Ah, I made the change too quick! Thanks for all the details here. I have moved these annotations to the CSV and removed the deprecated `operators.openshift.io/infrastructure-features` annotation.